python/ray/util/tracing/fp.py

```diff
@@ -0,0 +1,364 @@
+# from opentelemetry.instrumentation.fastapi import FastAPIInstrumentor
```

Comment:
FastAPIInstrumentor instrumentation was not working and was throwing errors so I added to add this.
This is taken from https://github.com/anyscale/tracing-example/blob/main/fp.py